mw-webapp/src/globalStore/ThemeStore.ts

```diff
@@ -40,6 +40,10 @@ const themedVariables: Record<string, Record<Theme, string>> = {
     [Theme.DARK]: "#686085",
     [Theme.LIGHT]: "#F2EFF9",
   },
+  hoverBgColor: {
+    [Theme.DARK]: "#A8A1DD",
+    [Theme.LIGHT]: "#A8A1DD",
+  },
```

Comment:
Do we have such variable in tokens?

Reply:
Yes
<img src="https://github.com/tritonJS826/masters-way/assets/8752900/27f185ad-18e8-438b-9535-16d4b2347cc6" width="200" />